Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -359,3 +359,54 @@ After adding paths to the autoload you should run `composer dumpautoload`. This
    :scale: 65 %
 .. |satis abandoned note| image:: ../Images/satis-abandoned.png
 .. |composer abandoned note| image:: ../Images/composer-ter-abandoned.png
+
+
+New file locations
+==================
+
+As final steps, you should move some files because the location will have
+changed for your site since moving to Composer.
+
+Move site configuration::
+
+   mv public/typo3conf/sites config/sites
+
+
+Move translation files::
+
+   mv public/typo3conf/l10n var/labels
+
+
+Some files are now no longer available inside the web root. For example:
+
+logs:
+
++---------+---------------------------+
+| After:  | var/log                   |
++---------+---------------------------+
+| Before: | public/typo3temp/var/log  |
```

Comment:
These two rows should be switched.

Also why not suggest moving these too?

It could also make sense to have a single table with all paths (before/after) and a general note to move the directories.

Reply:
Yes, good suggestion! I will make the change.